widgetssdk/src/main/java/com/glia/widgets/helper/ResourceProvider.java

```diff
@@ -78,6 +82,14 @@ public int convertDpToIntPixel(float dp) {
         return Math.round(convertDpToPixel(dp));
     }
 
+    @Override
+    public String getResourceKey(int stringKey) {
+        /*
+            getResourceName wil contain package and resourceName separated by "/", discarding the first package part
+         */
+        return weakContext.get().getResources().getResourceName(stringKey).split("/")[1];
+    }
```

Comment:
It seems to me that I have been reviewing many of these changes recently in some other PR, am I right?
I even remember I was suggesting to add resource name example to this comment.

Reply:
I think this branch needs to be rebased to the recent `feature/custom_locals` branch state.